code/datums/jobs.dm

```diff
@@ -2007,7 +2007,32 @@
 		..()
 		if (!M)
 			return
-		M.bioHolder.AddEffect("eyebeams")
+		M.traitHolder.addTrait("training_security")
+		if(prob(60))
+			var/aggressive = pick("eyebeams","cryokinesis")
+			var/defensive = pick("fire_resist","cold_resist","food_rad_resist","breathless") // no thermal resist, gotta have some sort of comic book weakness
+			var/datum/bioEffect/power/be = M.bioHolder.AddEffect(aggressive, magical=1)
+			if(aggressive == "eyebeams")
+				var/datum/bioEffect/power/eyebeams/eb = be
+				eb.stun_mode = 1
+			else
+				be.power = 1
+				be.altered = 1
+				be.safety = 1
```

Comment:
empowered, synchronized, stabilized genes that can't be cured by mutadone or scrambled away by genetics exceeds what I would consider "discount" that's more of an actual super hero